.gitignore

```diff
@@ -1,3 +1,6 @@
+.vscode/
+tmp/
+build/
```

Comment:
Please don't add these, smoltcp's `.gitignore` is not the place to collect the configuration files of every editor known to man. Please use a global gitignore on your system.